k8s/configure_ssl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+export VAULT_ADDR=https://vault.dide.ic.ac.uk:8200
```

Comment:
Is there a k8s/vault adaptor to make this easier? Not necessarily for this PR, but feels like something we might do a bunch of we do more with this style.

Reply:
il have a look into that... in the past when we stored secerts in azure key vault we would run and get the secrets in CI and create the secret that way so kubernetes could use it... but il see if there is other ways